src/audio/volume/volume.c

```diff
@@ -148,6 +149,35 @@ static enum task_state vol_work(void *data)
 	return SOF_TASK_STATE_COMPLETED;
 }
 
+/**
+ * \brief Allocates task of volume component.
+ * \param[in,out] dev Volume base component device.
+ * \return Error code.
+ */
+static int vol_task_init(struct comp_dev *dev)
+{
+	struct comp_data *cd = comp_get_drvdata(dev);
+	int ret = 0;
+
+	/* initialize task if necessary */
+	if (!cd->volwork) {
+		cd->volwork = rzalloc(SOF_MEM_ZONE_RUNTIME, 0, SOF_MEM_CAPS_RAM,
+				      sizeof(*cd->volwork));
```

Comment:
Could reduce one level of identation if checking the negated condition.

Reply:
@dbaluta Yes, you're right. Changed it.